drivers/driver.cmake

```diff
@@ -0,0 +1,63 @@
+
+if(CMAKE_COMPILER_IS_GNUCC)
+    set(LIBS_C_FLAGS -Wmissing-declarations -Wmissing-prototypes)
+endif(CMAKE_COMPILER_IS_GNUCC)
+
+if(CMAKE_COMPILER_IS_CLANG)
+    set(LIBS_C_FLAGS -Wmissing-declarations -Wmissing-prototypes -Wdocumentation -Wno-documentation-deprecated-sync -Wunreachable-code)
+endif(CMAKE_COMPILER_IS_CLANG)
```

Comment:
Setting warning flags seems to be about the quality and conventions of a particular part of the code, so it feels out of place in the generic file.

I'm not sure about the MSVC part below.

Reply:
I've reduced the scope of `driver.cmake` to builtin and generated test drivers (renamed builtin.cmake). With that scope reduction it is fine to keep this in this file.